### DISCUSSION TITLE
Shortcut to any menu item

### DISCUSSION THREAD

**Discussion by BLUESALOON** [egzumer/uv-k5-firmware-custom#410] [Ideas]
Firstly, thanks to all who contribute to this firmware.
Would it be possible to assign recall of a menu item to one of the programmable keys?
For example...  
If I want to access D.LIST I can either press MENU and scroll to it or I can press   MENU,  5,   4   to go to D.LIST directly.
Could a macro be created for those key presses.  MENU   1stDigit,    2ndDigit   (MENU,  5,   4) and be assigned to one of the side keys?
Would anyone else actually want that functionality?
Thanks

**Comment by jan-Sanku** [comment]
I think a programmable key option to open the D.LIST would be useful in general
